ui/src/app/utils/index.tsx

```diff
@@ -359,6 +360,11 @@ export const withCdrVersions = () => {
   return connectReplaySubject(cdrVersionStore, 'cdrVersionListResponse');
 };
 
+// HOC that provides a 'queryParams' prop with current query params
```

Comment:
nit: I don't know what HOC means, but if it's obvious to everyone else it's fine.

Reply:
https://reactjs.org/docs/higher-order-components.html but I'm mostly faking my knowledge